README.md

```diff
@@ -64,9 +97,12 @@ module.exports = {
 
 <!-- begin auto-generated configs list -->
 
-|    | Name          |
-| :- | :------------ |
-| ✅  | `recommended` |
+|    | Name              |
```

Comment:
If you add a `description` property to each config, it will show up in this table. Up to you if you think it would be helpful.